File: wlr/OGUR/OGUR/States/GameOverState.cs
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;
using SPX.States;
using SPX.Core;

namespace OGUR.States
{
    public class GameOverState : State
    {
        private readonly Texture2D m_menuBase;

        public GameOverState()
        {
            m_menuBase = XnaManager.GetGameOverAsset();
        }

        public void Draw()
        {
            var x = (XnaManager.WindowWidth - m_menuBase.Bounds.Right)/2;
            var y = (XnaManager.WindowHeight - m_menuBase.Bounds.Bottom) / 2;
            XnaManager.Renderer.Draw(m_menuBase, new Vector2(x,y), Color.White);
        }

        public void Update()
        {
            if(InputManager.IsPressed(Commands.Confirm,0,true))
            {
                StateManager.LoadState(new GameplayState());
            }
        }

        public void LoadContent()
        {

        }
    }
}

File: port-workspace/aigilas-ported/src/com/aigilas/states/GameOverState.java
package com.aigilas.states;import com.xna.wrapper.*;import java.util.*;import com.spx.states.*;import com.spx.core.*;import com.aigilas.management.*;import com.spx.io.*;
    public class GameOverState implements State
    {
        private Texture2D _menuBase;

        public GameOverState()
        {
            _menuBase = XnaManager.GetGameOverAsset();
        }

        public void Draw()
        {
            float x = (XnaManager.WindowWidth - _menuBase.Bounds.Right)/2;
            float y = (XnaManager.WindowHeight - _menuBase.Bounds.Bottom) / 2;
            XnaManager.Renderer.Draw(_menuBase, new Vector2(x,y), Color.White);
        }

        public void Update()
        {
            if(Input.IsActive(Commands.Confirm,0,true))
            {
                StateManager.LoadState(new GameplayState());
            }
        }

        public void LoadContent()
        {

        }
    }
